api/v1beta1/events.go

```diff
@@ -228,13 +234,13 @@ var Events = map[string]DisruptionEvent{
 	EventDisruptionCreated: {
 		Type:                        corev1.EventTypeNormal,
 		Reason:                      EventDisruptionCreated,
-		OnDisruptionTemplateMessage: "Disruption created",
+		OnDisruptionTemplateMessage: "disruption created",
 		Category:                    DisruptEvent,
 	},
 	EventDisruptionFinished: {
 		Type:                        corev1.EventTypeNormal,
 		Reason:                      EventDisruptionFinished,
-		OnDisruptionTemplateMessage: "Disruption finished",
+		OnDisruptionTemplateMessage: "disruption finished",
```

Comment:
Why lower case these? that isn't consistent throughout the file, afaict

Reply:
Good catch! It is a typo' error.